ingest/sqs/service/sqs_pool_tracker_test.go

```diff
@@ -0,0 +1,99 @@
+package service_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v24/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v24/ingest/sqs/service"
+)
+
+type SQSServiceTestSuite struct {
+	apptesting.ConcentratedKeeperTestHelper
+}
+
+func TestSQSServiceTestSuite(t *testing.T) {
+	suite.Run(t, new(SQSServiceTestSuite))
+}
+
+// This is a sanity-check test ensuring that all pools are tracked and returned correctly.
+func (s *SQSServiceTestSuite) TestPoolTracker_Track() {
+	s.Setup()
+
+	allPools := s.PrepareAllSupportedPools()
+
+	poolTracker := service.NewPoolTracker()
+
+	////////// Concetrated
+
+	// Get the concentrated pool
+	concentratedPool, err := s.App.ConcentratedLiquidityKeeper.GetConcentratedPoolById(s.Ctx, allPools.ConcentratedPoolID)
+	s.Require().NoError(err)
+
+	// Track the concentrated pool
+	poolTracker.TrackConcentrated(concentratedPool)
+
+	// Get the concentrated pool
+	concentratedPools := poolTracker.GetConcentratedPools()
+	s.Require().Len(concentratedPools, 1)
+
+	////////// CFMM
+
+	// Get balancer pools
+	balancerPool, err := s.App.PoolManagerKeeper.GetPool(s.Ctx, allPools.BalancerPoolID)
+	s.Require().NoError(err)
+
+	// Track the balancer pool
+	poolTracker.TrackCFMM(balancerPool)
+
+	// Track the same pool to ensure no duplicates
+	poolTracker.TrackCFMM(balancerPool)
+
```

Comment:
Why is this special to CFMM pool test and not the others

Reply:
This was added as a sanity check. Given the same implementation and low complexity, didn't feel necessary to add for all. Though it wouldn't hurt.

I ended up expanding assertions to other pools per your comment:
[fde8df6](https://github.com/osmosis-labs/osmosis/pull/7879/commits/fde8df6275fe2e30c215b31fc7d957388e068f31)